src/test/java/fi/otavanopisto/kuntaapi/test/AbstractIntegrationTest.java

```diff
@@ -248,6 +260,70 @@ protected String getClientSecret(AccessType accessType) {
     return String.format("%s_SECRET", accessType.name());
   }
   
+  private Long findClientDatabaseIdByAccessType(AccessType accessType) {
+    return findClientDatabaseIdByClientId(getClientId(accessType));
+  }
+  
+  private Long findClientDatabaseIdByClientId(String clientId) {
+    Connection connection = getConnection();
+    
+    try {
+      try {
+        connection.setAutoCommit(true);
+        PreparedStatement statement = connection.prepareStatement("select id from Client where clientId = ?");
+        try {
+          statement.setObject(1, clientId);
+          statement.execute();
+          ResultSet resultSet = statement.getResultSet();
+          if (resultSet.next()) {
+            return resultSet.getLong(1);
+          }
+        } finally {
+          statement.close();
+        }
+      } finally {
+        connection.close();
+      }
+    } catch (Exception e) {
+      fail(e.getMessage());
+    }
+    
+    return null;
+ }
+  
+  private Long findIdentifierIdByKuntaApiId(String kuntaApiId) {
+    Connection connection = getConnection();
+    
+    try {
+      try {
+        connection.setAutoCommit(true);
+        PreparedStatement statement = connection.prepareStatement("select id from Identifier where kuntaApiId = ?");
+        try {
+          statement.setObject(1, kuntaApiId);
+          statement.execute();
+          ResultSet resultSet = statement.getResultSet();
```

Comment:
![BLOCKER](https://sonarsource.github.io/sonar-github/severity-blocker.png 'Severity: BLOCKER') Use try-with-resources or close this "ResultSet" in a "finally" clause. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS2095)